homeassistant/components/islamic_prayer_times/__init__.py

```diff
@@ -29,6 +29,16 @@ def update_unique_id(
 
     await er.async_migrate_entries(hass, config_entry.entry_id, update_unique_id)
 
+    # add lat and lon to entry data if not present
+    if not config_entry.data:
+        data = {
+            CONF_LOCATION: {
```

Comment:
Please avoid putting complex objects in the config entry data

Reply:
Please check the latest commit.